app/scripts/modules/cloudfoundry/src/pipeline/stages/unshareService/cloudfoundryUnshareServiceStage.module.ts

```diff
@@ -0,0 +1,36 @@
+import { IController, IScope, module } from 'angular';
+import { react2angular } from 'react2angular';
+
+import { CloudfoundryUnshareServiceStageConfig } from './CloudfoundryUnshareServiceStageConfig';
+import { ExecutionDetailsTasks, IStage, Registry } from '@spinnaker/core';
+import { CloudfoundryServiceExecutionDetails } from 'cloudfoundry/presentation';
+
+class CloudFoundryUnshareServiceStageCtrl implements IController {
+  public static $inject = ['$scope'];
+  constructor(public $scope: IScope) {}
+}
+
+export const CLOUD_FOUNDRY_UNSHARE_SERVICE_STAGE = 'spinnaker.cloudfoundry.pipeline.stage.unshareServiceStage';
+module(CLOUD_FOUNDRY_UNSHARE_SERVICE_STAGE, [])
```

Comment:
Look to `waitStage` for an example of why we don't need to wrap this in an angular component anymore.

Reply:
👍